robottelo/ui/base.py

```diff
@@ -220,6 +242,31 @@ def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
             )
             return None
 
+    def wait_until_element_is_clickable(self,
+                                        locator,
+                                        timeout=12,
+                                        poll_frequency=0.5):
+        """Wrapper around Selenium's WebDriver that allows you to pause your
+        test until an element in the web page is present and can be clicked.
+
+        """
+        try:
+            element = WebDriverWait(
+                self.browser, timeout, poll_frequency
+            ).until(expected_conditions.element_to_be_clickable(locator))
+            self.wait_for_ajax(poll_frequency=poll_frequency)
+            if element.get_attribute('disabled') == u'true':
+                return None
+            return element
+        except TimeoutException as err:
+            self.logger.debug(
+                "%s: Timed out waiting for element '%s' to display or to be "
+                "clickable.",
+                type(err).__name__,
+                locator[1]
+            )
+            return None
```

Comment:
Not sure about question for ```wait_until_element_exists``` as it is most elementary verification and will return ```None``` only if element is not present in DOM.
Next level is ```wait_until_element``` method which will return ```None``` only if element is not present and not visible on the page. These both condition have to be true in ascending order as element cannot be visible and not be present on the page (of course it can be present, but not visible, but that conditions can be verified in ```wait_until_element_exists``` method, but ```wait_until_element``` should verify things mentioned above)